SparkPost/matchbox
# UX-747 & UX-748: Add Style Dictionary and CSS custom properties file

### What Changed
- Adds style dictionary, with a basic setup
- Adds all tokens in JSON format

Notes:
This PR makes some decisions on token structure, which affects how tokens are transformed and formatted. We may need to adjust later on in order to accommodate our custom formats.

### How To Test or Verify
- Pull down this repository and run `npm run build`
- Verify the `dist` folder is generated correctly with all tokens

### PR Checklist
